bears/yaml/TravisLintBear.py

```diff
@@ -24,6 +26,30 @@ class TravisLintBear:
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
+    CHECK_CONNECTION_URL = 'https://travis-ci.org/'
+
+    @classmethod
+    def check_prerequisites(cls):
+        base_check = super().check_prerequisites()
+        if base_check is not True:
+            return base_check
+        url_status = cls.get_url_status(cls.CHECK_CONNECTION_URL)
+
+        try:
+            if url_status is None:
+                return 'You are not connected to the internet.'
+            else:
+                url_status.raise_for_status()
+                return True
+        except requests.exceptions.HTTPError:
+            return 'Failed to establish a connection to the server.'
```

Comment:
"which server?" I would ask myself now if I get the message :) You maybe want to include the url to travis: `CHECK_CONNECTION_URL`